### ISSUE TITLE
As a user, I want to know what's behind bucoli.co so I can learn.

### ISSUE THREAD

**Issue by vjpixel** [memeLab/bucoli.co#83]
_Deadline: Mon 12:00_

Publishing LinkedIn articles is both a way to document our trajectory and a way to increase interest in bucoli.co.

We want to have a making-of article published.

**Comment by vjpixel** [comment]
My plan was to create an article about "how I created bucoli.co roadmap". But after creating the roadmap, I'm not secure to write an article. I think I need to learn more about roadmaps and/or improve our roadmap before writing this article.

Since we're still not advertising bucoli.co, I decided to not run for a subject to write about now. I'll wait for our advertising strategy to start being implemented. This will (1) let us measure the impact of the "making of" itself and (2) allow people to be curious about what we're creating _before_ showing how we're creating it.

**Comment by vjpixel** [comment]
@tiagobugarin, please tell me if you agree.

**Comment by tiagobugarin** [comment]
yes, agree. better wait and collect your thoughts and metrics and have a clear undestanding of everything bucoli.co and then write about it.
